invenio_jobs/registry.py

```diff
@@ -34,11 +34,12 @@ def get_job_id(self, instance):
                 return job_id
         raise KeyError("Job not found in registry.")
 
-    def all_registered_jobs(self):
+    def get_all(self):
```

Comment:
similary this could be a property as well i.e. all methods of `def method(self)`